src/services/sectionService.ts

```diff
@@ -133,3 +134,17 @@ export const getTagsToAdd = async (tags: Tag[], context: MyContext, sectionId: n
 
   return Array.isArray(tagsToAdd) ? tagsToAdd : [];
 }
+
+export const getTagsToRemove = async (tags: Tag[], context: MyContext, sectionId: number): Promise<SectionTag[]> => {
+  //Get all the existing tags associated with this section in SectionTags
+  const existingTags = await SectionTag.getSectionTagsBySectionId('updateSection resolver', context, sectionId);
```

Comment:
prefer the `reference` here to be relevant to the service since it could technically be called from outside the resolver workflow.

The references are intended to help us quickly identify where an issue happened once we've got all our logs going into OpenSearch.